metrics/cli/docs/05_user_manual_steps.md

```diff
@@ -0,0 +1,127 @@
+2. Navigate to the Sign-in method's tab on the Firebase auth page or use the following link: https://console.firebase.google.com/project/project_id/authentication/providers
+3. Enable an Email/Password provider.
+4. Enable a Google provider.
+5. On the Google provider page, open the Web SDK configuration tab, copy the Web client ID and paste it to the console
+
+Paste your Web client Id here:
+```
+### Firebase Analytics configuration
+
+Since the Metrics Project using Firebase Analytics and the analytics is disabled by default, we should provide a detailed instruction,
+which helps the user to configure the Firebase analytics during the Metrics deployment process using the Metrics CLI.
+
+Here is a prompt instruction, which helps the user to configure Firebase analytics easily:
+
+```text
+Firebase analytics configuration instruction:
+
+To enable Firebase analytics for the current deploying project, consider the following steps:
+
+1. Follow the link and click an "Enable Google Analytics" button: https://console.firebase.google.com/project/project_id/settings/integrations/analytics
+
+Press any key to continue:
+```
+### Default region configuration
+
+Since the deployment of the Metrics project requires a GCloud project's region, we should provide a detailed instruction, 
+which helps the user to select the [region/zone](https://cloud.google.com/compute/docs/regions-zones) during the Metrics deployment process using the Metrics CLI.
+
+Here is a prompt instruction, which helps the user to select a GCloud project's region:
+
+```text
+Here is a list of available regions for gcloud project:
+
+REGION                   SUPPORTS STANDARD  SUPPORTS FLEXIBLE
+asia-east2               YES                YES
+asia-northeast1          YES                YES
+asia-northeast2          YES                YES
+asia-northeast3          YES                YES
+asia-south1              YES                YES
+asia-southeast2          YES                YES
+...............          ...                ...
+
+Please enter the region name that suits you best:
+```
+### Initial Firestore Data configuration
+
+Since the Metrics project controls the allowed email domains and features config using the Firestore database appropriate collections, 
+we should provide a detailed instruction, which helps the user to initialize the Firestore database with the mentioned collections' data during the Metrics deployment process using the Metrics CLI.
+
+Here is a prompt instruction, which helps the user to initialize the Firestore database:
+
+```text
+Initialize Firestore data instruction:
+
+To initialize Firestore data for the current deploying project, consider the following steps:
+
+1. Follow the link to open the Firestore database: https://console.firebase.google.com/project/project_id/firestore
+2. Create a collection with the "allowed_email_domains" id.
+3. Create document(s) with the "domain" id without any fields.
```

Comment:
It would be great to provide a link explaining the process of adding an allowed email domain in more detail. Also, the link explaining the structure of the `feature config` database document would help the user create it properly.